.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,55 @@
+<!--- ↑ Provide a general summary of your changes in the Title above ↑ -->
+
+## Description
+
+<!--- Describe your changes in detail -->
+
+## Related Issue
+
+<!--- This project only accepts pull requests related to open issues -->
+<!--- If suggesting a new feature or change, please discuss it in an -->
+<!--- issue first -->
+<!--- If fixing a bug, there should be an issue describing it with -->
+<!--- steps to reproduce -->
+<!--- Please link to the issue here with the prefix or resolve -->
+<!--- keyword -->
```

Comment:
This is not needed, as the bot link it through the task ID from the branch's name.

Reply:
Can be cool for reviewers to have a link?

Reply:
The link will be added automatically by the bot
![image](https://user-images.githubusercontent.com/31956107/117166229-7865cf00-adc6-11eb-884a-cbda482de4cb.png)

Reply:
True. I change it 👍